recipes-bsp/u-boot/u-boot-starfive_v2021.10.bb

```diff
@@ -10,7 +10,7 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 BRANCH:visionfive2 = "JH7110_VisionFive2_devel"
 BRANCH:star64 = "Star64"
 
-SRC_URI = "git://github.com/starfive-tech/u-boot.git;protocol=https;branch=${BRANCH} \
+SRC_URI = "git://github.com/starfive-tech/u-boot.git;protocol=https;nobranch=1 \
```

Comment:
Thanks for looking into it. I think it would be nicer if we bumped the SRCREV to `JH7110_VF2_6.1_v5.10.3` tag equivalent SHA1 which is 8c7b3f31fb546f829bbce9ee52435342314fabbf . This would be inline with kernel and other BSP component that were bumped recently.

Reply:
Thanks - I will do that and re-submit.

Reply:
OK, I updated the has to be from tag JH7110_VF2_6.1_v5.10.3
I built and tested this on my vf2 board.